setup.py

```diff
@@ -28,8 +28,7 @@
     jax_version = [line[4:].strip() for line in f.readlines() if "jax=" in line][0]
 
 requirements = [
-    # TODO: Change to "pennylane>=0.31" when next version releases.
-    "pennylane @ git+https://github.com/PennyLaneAI/pennylane/@master",
+    "pennylane>=0.31",
```

Comment:
Are there any other places we will need to make changes like this?

Reply:
No, that should be it.